app/forms/Ducr.scala

```diff
@@ -21,6 +21,7 @@ import play.api.data.Forms.{mapping, text}
 import play.api.data.validation.Constraints.pattern
 import play.api.libs.json.Json
 
+// TODO change argument name from 'ducr' to 'value', this change is incompatible with backend
```

Comment:
Should this be done now?

Reply:
Why rename at all?  There are a number of "form" classes with single elements with the same name as the class.  If we "fix" this we should fix them all.

Reply:
I'm removing this TODO